frontend/src/components/page-new/sunday-celebration.js

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { SecondaryButtonLink } from "../Button";
+import {containerLg, timeLg, buttonLg, titleLg, locationLg} from "./sunday-celebration.module.css"
+
+const SundayCelebrationSection = () => {
+  return (
+    <div className="bg-Neutral-200 w-full flex justify-center items-center p-6 lg:py-16">
+      <div className={`flex flex-col bg-Shades-0 items-center h-full border-solid border-2 max-w-container rounded-xl py-6 px-10 border-Neutral-700 lg:px-16 lg:py-10 ${containerLg}`}>
```

Comment:
Because the order of objects changes in mobile vs desktop mode, this element is a flexbox in mobile mode and a CSS grid in desktop mode (grid styles imported from the `.module.css` file. Let me know if there's a better way of doing this!

Reply:
you could put this all in the module.css file.  no need to separate it.